<launch>
  <arg name="jsX" default="0"/>
  <!-- Drive Control Nodes -->
  <node name="joystick_controller" pkg="joy" type="joy_node">
    <param name="autorepeat_rate" type="double" value="30"/>
    <param name="joystick_controller/dev" value="/dev/input/js$(arg jsX)"/>
  </node>
  <node pkg="rover_5_navigation" type="base_controller_i2c.py" name="base_controller_i2c"/>
</launch>
